src/System.Runtime.Serialization.Json/tests/System.Runtime.Serialization.Json.Tests.csproj

```diff
@@ -10,29 +10,36 @@
     <RootNamespace>System.Runtime.Serialization.Json.Tests</RootNamespace>
     <AssemblyName>System.Runtime.Serialization.Json.Tests</AssemblyName>
     <ProjectGuid>{701CB3BC-00DC-435D-BDE4-C5FC29A708A7}</ProjectGuid>
+    <UseProjectReferences>true</UseProjectReferences>
+  </PropertyGroup>
+  <PropertyGroup Condition="!$(UseProjectReferences)" >
+    <ProjectJson>Performance/project.json</ProjectJson>
```

Comment:
To clarify regarding project.lock.json, we need to have the project.lock.json entry in the csproj in order build successfully. Otherwise the original project.lock.json is used and errors will be thrown due to types not found. We're not checking in any project.lock.json files.